notebooks/Interactive_Dashboard.ipynb

```diff
@@ -0,0 +1,724 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import panel as pn\n",
+    "import panelify\n",
+    "\n",
+    "pn.extension()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "## Accessing the Data (Plots)\n",
+    "\n",
+    "The plots from this example are stored on the\n",
+    "[CGD webext machine](https://webext.cgd.ucar.edu/), specifically from\n",
+    "[this specific case](https://webext.cgd.ucar.edu/g.e22b05.G1850ECOIAF_JRA.TL319_g17.cocco.001/),\n",
+    "which we use as the path.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path = (\n",
+    "    \"https://webext.cgd.ucar.edu/g.e22b05.G1850ECOIAF_JRA.TL319_g17.cocco.001/\"\n",
```

Comment:
Can you change the path to https://webext.cgd.ucar.edu/g.e22.G1850ECO_JRA_HR.TL319_t13.004/ instead of the current selection? Of the three runs we want to look at for the high-res analysis, I think 004 is the natural default option.

To elaborate, the three cases are

1. https://webext.cgd.ucar.edu/g.e22.G1850ECO_JRA_HR.TL319_t13.004/ -- The 0.1 degree ocean / ice simulation we are running for a full JRA cycle (61 years; we're currently ~halfway there)
2. https://webext.cgd.ucar.edu/g.e22.G1850ECO_JRA_HR.TL319_t13.003/ -- A 0.1 degree ocean / ice simulation with initial conditions that are slightly different from 004*; we ran for four years, compared to the first four years of 004, and decided to continue the latter
3. https://webext.cgd.ucar.edu/g.e22b05.G1850ECOIAF_JRA.TL319_g17.cocco.001/ -- A 1 degree ocean / ice simulation that provided initial conditions for our high-res run (we interpolated the BGC tracers onto the 0.1 degree grid); we don't have a full cycle of output from the run, but it's a good control experiment for comparing the high resolution run to the 1 degree resolution.

So it's nitpicky, but i think 004 is the best case to use as an example highlighting what the dashboard can do.

\* If you're curious about the difference between 003 and 004, initial conditions for all BGC tracers from 004 were interpolated from the `cocco.001` case, while 003 used some of those initial conditions and World Ocean Atlas observations for others. There weren't significant differences between the two cases after four years, so we went with 004 because the consistency among initial conditions was preferable